packages/hooks/src/useSubscription.ts

```diff
@@ -36,6 +36,7 @@ export function useSubscription<TData = any, TVariables = OperationVariables>(
   subscriptionData.context = context;
 
   useEffect(() => subscriptionData.afterExecute());
+  useEffect(() => subscriptionData.cleanup.bind(subscriptionData), []);
```

Comment:
if i am not mistaken, this needs to be invoked? `subscriptionData.cleanup.bind(subscriptionData)()`